src/tests/mod.rs

```diff
@@ -76,7 +76,8 @@ macro_rules! generate_tests {
         glue!(ordering, ordering::ordering);
         glue!(sql_types, sql_types::sql_types);
         glue!(synthesize, synthesize::synthesize);
-        glue!(unique, unique::unique);
+        glue!(validate_unique, validate::unique::unique);
+        glue!(validate_types, validate::types::types);
```

Comment:
Wow, this is cool. Thanks!

Reply:
Did the same thing with #144, I figured the files were getting a bit messy :-p

Reply:
Wow, yup that sounds nice :) thanks!